### PR TITLE
Keep cjk characters unescaped.

### PR DESCRIPTION
**Pull request by bcho**
Instead of showing escaped characters in editing mode:

```json
{
    "project": "a-cool-project",
    "start": "2015-10-19 13:27:11",
    "stop": "2015-10-19 14:53:08",
    "tags": [
        "\u91cd\u6784"
    ]
}
```

Escaped characters should be:

```json
{
    "project": "a-cool-project",
    "start": "2015-10-19 13:27:11",
    "stop": "2015-10-19 14:53:08",
    "tags": [
        "重构"
    ]
}
```

Which can give a better look.

### CODE REVIEW

**Comment by bcho** [comment]
@TailorDev Should we also unescaped the cjk characters when saving the frames? https://github.com/TailorDev/Watson/blob/develop/watson/watson.py#L144

**Comment by jmaupetit** [comment]
@bcho thank you for this!
@k4nar WDYT?

**Comment by jmaupetit** [comment]
@bcho I think you are right: we should unescape cjk characters when saving. Can you add this to this PR?

**Comment by bcho** [comment]
@jmaupetit Patch updated!

**Comment by willdurand** [comment]
Do we have tests for that?

**Comment by bcho** [comment]
@willdurand nope, I think this should be tested by hand.

**Comment by k4nar** [comment]
:+1: 

I think we should have a test case `save_frame_with_cjk_char` though. You will find all the tests for the `save` function here: https://github.com/TailorDev/Watson/blob/develop/tests/test_watson.py#L319

**Comment by bcho** [comment]
@k4nar @willdurand ok, I will submit test later :)

File: tests/test_watson.py [hunk 1]
```diff
@@ -353,6 +353,9 @@ def test_save_current_without_tags(watson):
             assert isinstance(result['start'], (int, float))
             assert result['tags'] == []
 
+            dump_args = json_mock.call_args[1]
+            assert dump_args['ensure_ascii'] is False
```
Comment (bcho) [inline tests/test_watson.py]: @k4nar @willdurand Is it ok to test calling params only? I don't figure out how to check the dumped json value.

Comment (k4nar) [inline tests/test_watson.py]: Yeah, as we mock the json call we can't really check the output.

**Comment by k4nar** [comment]
LGTM, the test case could be more thorough but we were already UTF-8 compliant, so I guess it's fine.

**Comment by jmaupetit** [comment]
:+1:

**Pull request merged**